common/util.js

```diff
@@ -96,6 +96,27 @@ export function setIn(obj, path, value) {
   return obj
 }
 
+
+/**
+ * Returns an array of [key, val] for all
+ * keys in provided object where fn(key) returns true.
+ *
+ * @param obj object to walk
+ * @param fn will be passed key. is expected to return true iff key/ should be talken.
+ * @param result don't use
+ */
+export function findDeepInObj(obj, fn, result = []) {
```

Comment:
Is it accepted in JS to have parameters like `result` that are not to be used by callers? (You can see my total ignorance of all thing JavaScript...)